scripts/update-repo-internal.sh

```diff
@@ -16,17 +16,20 @@ RELEASE="${DISTRO##*-}"
 case $PARENT in
     centos)
         mkdir -p repo/$PARENT/$RELEASE/packages
-        find package/$DISTRO/ -name *.rpm | \
+        find package/$DISTRO/ -name i'*.rpm' | grep -v lstore-release.rpm | \
```

Comment:
Man, using the "reply to comment via email" feature is an absolute catastrophe. If you're gonna do that, please trim the mail.